### PR TITLE
Add 'kalk' and 'clippo'

### PR DESCRIPTION
**Pull request by monsterkodi**
Clipboard Tool
https://github.com/monsterkodi/clippo

clippo
clippo

A(nother :) Simple Calculator
https://github.com/monsterkodi/kalk

kalk

### CODE REVIEW

File: apps/clippo/clippo.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: kalk
```
Comment (zeke) [inline apps/clippo/clippo.yml]: This should be `Clippo`, right?

Comment (monsterkodi) [reply apps/clippo/clippo.yml]: Hi zeke, 

not really, I like it lower case, but if necessary I can live with the upper case as well.

Comment (monsterkodi) [reply apps/clippo/clippo.yml]: Oops. sorry. Misunderstood your question. Yes, in the clippo.yaml it should be clippo instead of kalk.
I will fix that.

Comment (monsterkodi) [reply apps/clippo/clippo.yml]: Ok, should be fixed now. 
Thanks for the hint.

File: apps/clippo/clippo.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+screenshots:
+  -
+    imageUrl: 'https://raw.githubusercontent.com/monsterkodi/clippo/master/img/readme.png'
+    caption: 'windows'
+    imageLink: 'https://github.com/monsterkodi/clippo'
+  -
+    imageUrl: 'https://raw.githubusercontent.com/monsterkodi/clippo/master/img/shot.png'
+    caption: 'mac'
+    imageLink: 'https://github.com/monsterkodi/clippo'
+
```
Comment (codebytere) [inline apps/clippo/clippo.yml]: remove this newline

**Pull request merged**